packages/scxa-marker-gene-heatmap/src/MarkerGeneHeatmap.js

```diff
@@ -105,7 +109,11 @@ const MarkerGeneHeatmap = (props) => {
       value: plotLineAxisPosition,
       zIndex: zIndex,
       label: heatmapType !== `cellmultiexperiment` && {
-        text: splitCellTypeLabel.join(`<br/>`),
+        useHTML: true,
+        text:  checkIfAnnDataExperiment(splitCellTypeLabel) ?
+            `<img src="https://upload.wikimedia.org/wikipedia/commons/6/6a/External_link_font_awesome.svg" alt="external_link_image" width="11px"/>` +
+            splitCellTypeLabel.join(`<br/>`) :
+            splitCellTypeLabel.join(`<br/>`),
```

Comment:
This bit shouldn’t need changing since [we agreed to remove the experiment accessions from the right margin in the multiexperiment cell type heatmap](https://www.pivotaltracker.com/story/show/179903243). We had a regression there and it has been fixed in https://github.com/ebi-gene-expression-group/atlas-components/pull/115.